BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java

```diff
@@ -95,6 +102,48 @@ public void update(Observable arg0, Object arg1) {
       return;
     }
 
+  // Hook to write Notifications to a temporary file on the device/emulator
+    // Keeps track of incoming notifications and makes sure count is right
+    count++ ;
```

Comment:
if 100 lines is enough for your debugging, then that's fine with me. Have you considered re-creating the file from scratch every run?  is that too much?